flink-table/flink-sql-client/conf/sql-client-defaults.yaml

```diff
@@ -78,6 +78,15 @@ catalogs: [] # empty list
 #    hive-conf-dir: /opt/hive_conf/
 #    default-database: ...
 
+#==============================================================================
+# Modules
+#==============================================================================
+
+# Define modules here.
+
+#modules: # note the following modules will be of the order they are specified
```

Comment:
should here be `modules: [] # empty list `
like above catalogs

Reply:
not really. that would wipe out the default core module

Reply:
ok